critiquebrainz/frontend/views/moderators.py

```diff
@@ -0,0 +1,11 @@
+from flask import Blueprint, render_template
+from critiquebrainz.data.model.user import User
+from custom_config import ADMINS
```

Comment:
There may be ([and are](https://github.com/metabrainz/critiquebrainz/blob/master/critiquebrainz/frontend/__init__.py#L14-L28)) other config files. Some might exist, some might not. So the right thing to do here would be to use the `config` object from [`flask.current_app`](http://flask.pocoo.org/docs/0.12/api/#flask.current_app). You should be able to find examples throughout other CritiqueBrainz modules.